spec/api/v1/teams_spec.rb

```diff
@@ -131,10 +131,27 @@
 
     subject { get "/api/v1/teams/#{team.slug}/users", params: { per_page: 2, api_key: api_token.token } }
 
-    it 'returns 2 team users' do
-      subject
+    context 'when user team returns' do
+      it 'returns 2 team users' do
+        subject
 
-      expect(JSON.parse(response.body).count).to eq(2)
+        expect(JSON.parse(response.body).count).to eq(2)
+      end
+    end
+
+    context 'when no user team returns' do
```

Comment:
`when team does not return any user`

Reply:
because there is no team in the parameter (`params[:slug]`)